crates/pallets/starknet/src/genesis_loader.rs

```diff
@@ -50,7 +55,12 @@ impl<T: crate::Config> From<GenesisLoader> for GenesisConfig<T> {
                 let hash = unsafe { std::mem::transmute::<ClassHash, ClassHashWrapper>(hash) };
                 match class {
                     ContractClass::Path { path, version } => {
-                        (hash, get_contract_class(&read_file_to_string(&path), version))
+                        let contract_path = if loader.madara_path.is_some() {
```

Comment:
check if `contract_path == ""`? Or return an error from `get_project_path` if it doesn't find the project path

Reply:
What I mean is that in the current implementation, if madara_path is None and there is no project path, it will panic in `read_file_to_string`. Would be cleaner to check what is the value of contract_path and panic with some message if it is not correct (i.e. here should probably panic if `utils::get_project_path()` returns "", since that means madara_path is None and you couldn't find the project path).

Reply:
Okay 👍

Reply:
@greged93 can u check the latest commit?